api/types.proto

```diff
@@ -277,6 +277,41 @@ message UpdateConfig {
 
 	// Amount of time between updates.
 	Duration delay = 2 [(gogoproto.nullable) = false];
+
+	enum OnFailure {
+		PAUSE = 0;
```

Comment:
A related question is how to deal with concurrent rollouts. Should a rollout replace previous rollout, or be rejected if service is under rollout? (`Pause` implies this rollout is on-hold. It has not finished.)